### PR TITLE
Prev/next button

### LINKED ISSUE #68: Feat: Add prev and next button in `start-contribution` docs section
### Description

add the prev and next button to navigate between the previous and next contribution guide section

### Screenshots

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Aadarsh805**
## Fixes Issue
closes #68

## Changes proposed
Added previous and next buttons to the bottom of docs pages for navigating between the docs

## Screenshots

image

### CODE REVIEW

**Comment by Aadarsh805** [comment]
@sboy99 I made icon files for prev and next icon, but they are not showing for some reason, do check that issue
I'm not sure if I'm using the correct icons

**Comment by sboy99** [comment]
Fixed the issue of svgs

File: src/components/icons/NextIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ComponentProps } from 'react';
```
Comment (sboy99) [inline src/components/icons/NextIcon.tsx:1]: no need to keep it. remove it

File: src/components/icons/PrevIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { ComponentProps } from 'react';
```
Comment (sboy99) [inline src/components/icons/PrevIcon.tsx:1]: not in use, so we should remove it

**Review by sboy99 — CHANGES_REQUESTED** [review]
Super logic. really awesome, only few changes. Then we are good to 🚀

**Review by sboy99 — APPROVED** [review]
Great Work 🎉 . LGTM

**Pull request merged**